documentation/src/main/asciidoc/user_guide/clustering.adoc

```diff
@@ -1013,6 +1013,12 @@ Currently Infinispan provides the following implementations of EntryMergePolicy:
 
 | MergePolicies.REMOVE_ALL
 | Always remove a key from the cache when a conflict is detected.
+
+| MergePolicies.NONE
+| There is no action for conflicting entries. Entries are deleted on nodes which are not (longer) and owner. Copied from the primary owner to owners which does not have it, and not changed at all for nodes which are still owner and have the entry present no matter what value.
```

Comment:
I think this would be better as:

> No attempt is made to resolve conflicts. Instead, entries hosted on the minority partition are removed and the nodes in this partition do not hold any data until the rebalance starts. Note, this behaviour is equivalent to prior Infinispan versions which did not support conflict resolution.